packages/ppx/src/declarations_to_emotion.re

```diff
@@ -2093,7 +2093,24 @@ let text_transform =
   );
 let white_space =
   variants(Parser.property_white_space, (~loc) => [%expr CssJs.whiteSpace]);
-let tab_size = unsupportedProperty(Parser.property_tab_size);
+
+let render_tab_size = (~loc, value: Types.property_tab_size) => {
+  switch (value) {
+  | `Number(n) =>
+    int_of_float(n) < 0
+      ? raise(Invalid_value("tab-size value can not be less than 0!"))
```

Comment:
This error doesn't happen at code-generation, it happens already at parsing. No need to raise anything/check for it

Reply:
I don't see how it shall error at parsing unless if we have bounds, i.e `... [0,∞]`, which IIRC, is not supported at the moment.

Reply:
If we want to have this kind of validation we would need to create a `let nonnegative-number` on Parser.re (but that's out of the scope from this)

Reply:
Right, in that case, do you give this snippet a green light?